Spigot-Server-Patches/0551-Optimize-Villagers.patch

```diff
@@ -93,25 +102,41 @@ index 35eb3a5a61145e94d5b0c77c0eb13bfa46fac23b..6861b1a345496a83900b0ef702ba3431
 +                }
 +            }
 +        }
-+        // Clean up - vanilla does this only when it runs out, but that would push it to try farther POI's...
-+        this.e.long2LongEntrySet().removeIf((entry) -> entry.getLongValue() < this.d);
++
+         this.f.long2ObjectEntrySet().removeIf((entry) -> {
+-            return !((BehaviorFindPosition.a) entry.getValue()).b(i);
++            return entry.getValue().b < e;
```

Comment:
why was this changed?

Reply:
To fit with your changes. See your original filtering (entry.getLongValue() is now entry.getValue().b)